### DISCUSSION TITLE
Literature and sources at the end

### DISCUSSION THREAD

**Discussion by treeandcoffee** [GetPublii/Publii#1864] [General] [answered]
Hi
How do you solve this on a website/blog so that many sources are listed at the end of the article or page? This is particularly necessary in the science sector. Thx.

**Comment by internettips** [comment, marked as answer]
Try using the Footnotes plugin from: [https://p4p.pages.dev/plugin-footnotes/](url) . More at: [https://github.com/GetPublii/Publii/discussions/1334](url)

**Reply by necs2010** [reply]
For some reason these are giving 404 errors. Do you have an update?

**Reply by bttrx** [reply]
@necs2010 The URL in both links is wrong. Try these instead: https://p4p.pages.dev/plugin-footnotes/ and https://github.com/GetPublii/Publii/discussions/1334

**Comment by treeandcoffee** [comment]
Absolutely great!! Many thanks for your answer.

**Comment by necs2010** [comment]
This is fantastic. This is just for the markdown editor, right? 
----
Nathaniel
